starter/script.js

```diff
@@ -1 +1,131 @@
-// put code here!
+var username = prompt("Hello there! You've been given a chance to go back in time to meet with some of modern history's most important political and popular figures. Before you meet them, you'll need to answer a few questions, and do some preperation for your meeting. Are you all set? What is your name");
+
+var year = prompt("Hello, " + username +  " Okay, let's select our destination. We have a few options. Enter the decade you'd like to visit: (1950/2030)")
+    if (year === "1950") {
+      var year = prompt("Heading to the 50s! Interesting time. Would you like to have lunch with President Harry Truman or Frank Sinatra? (T/S)")
+    } else if (year === "2030") {
+      prompt("Just kidding! This hasn't happened yet! Why not try 1950?")
+  } while (year != "1950" || "2030") {
+  	prompt("Please select a valid choice!")
+  }
+
+  
+//Options for first question for 1950//
+if (year  === "T") {
+	var year = prompt("Excellent. President Truman would like to Prep for the meeting. Would you like to discuss domestic or international affairs? (D/I)");
```

Comment:
Redeclaring the variable `year` several times with `var` will work, but it's considered cleaner to only use var once for each variable. This would be a good opportunity to declare all the variables at the top `var year;` and then each subsequent assignment can drop the `var` keyword.